Pod/Classes/SimplePing.m

```diff
@@ -627,13 +627,23 @@ - (void)startWithHostAddress {
         
         // The socket will now take care of cleaning up our file descriptor.
         
-        assert( CFSocketGetSocketFlags(self.socket) & kCFSocketCloseOnInvalidate );
+        CFSocketSetSocketFlags(self.socket, kCFSocketCloseOnInvalidate);
```

Comment:
Here for no reason you're erasing all default flags, specifically kCFSocketAutomaticallyReenableReadCallBack, which leads to issues with simultanious instances of SimplePing (#20)

Reply:
```suggestion
        CFSocketSetSocketFlags(self.socket, (CFSocketGetSocketFlags(self.socket) | kCFSocketCloseOnInvalidate));
```